### PR TITLE
All Hail the 1.0

### PR DESCRIPTION
**Pull request by Reqrefusion**
I'm adding 1.0 to the relevant places on the site. Actually, I waited for the artifacts to be created, but I couldn't stand sleeping, so I gave possible links to the relevant links. But considering the way it was created, it's probably okay. I guess there is nothing I missed.

If there are errors in the links, please change them. You can follow the process [here](https://github.com/FreeCAD/FreeCAD/releases/tag/1.0.0).

### CODE REVIEW

**Comment by adrianinsaval** [comment]
Links are not correct, they all should be FreeCAD_* not FreeCAD-*

File: downloads.php [hunk 1]
```diff
@@ -80,8 +80,8 @@ function thankyou(e) {
           <div class="card-body d-block align-items-center text-center px-xl-5 py-xl-4">
             <img class="w-100 p-4" src="svg/icon-linux.svg" alt="Linux">
             <h3 class="card-title download-platform-name m-0 pb-3">Linux</h3>
-            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/0.21.2/FreeCAD-0.21.2-Linux-x86_64.AppImage">x86_64 AppImage</a>
-            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/0.21.2/FreeCAD-0.21.2-Linux-aarch64.AppImage">aarch64 AppImage</a>
+            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD-1.0.0-conda-Linux-x86_64-py311.AppImage">x86_64 AppImage</a>
```
Comment (chennes) [inline downloads.php:83]:
```suggestion
            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD_1.0.0-conda-Linux-x86_64-py311.AppImage">x86_64 AppImage</a>
```

File: downloads.php [hunk 2]
```diff
@@ -80,8 +80,8 @@ function thankyou(e) {
...
+            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD-1.0.0-conda-Linux-aarch64-py311.AppImage">aarch64 AppImage</a>
```
Comment (chennes) [inline downloads.php:84]:
```suggestion
            <a class="btn btn-primary rounded-pill my-1" onclick="thankyou(event)" role="button" href="https://github.com/FreeCAD/FreeCAD/releases/download/1.0.0/FreeCAD_1.0.0-conda-Linux-aarch64-py311.AppImage">aarch64 AppImage</a>
```

**Comment by luzpaz** [comment]
Screenshot_20241119_113801


'See What's New' link is out-of-date!

**Pull request merged**